reports/TC39/2016-09.md

```diff
@@ -8,6 +8,36 @@ Before moving forward, I'd like to repeat the same statement from [my first repo
+1. [Unify String and Array maximum lengths](https://github.com/tc39/ecma262/pull/641) Defer to November 2016 (Michael Saboff)
+1. [Standardize Date.UTC when called with 1 argument](https://github.com/tc39/ecma262/pull/642) (Brian Terlson)
+1. Retracting [ArrayBuffer.transfer](https://gist.github.com/lukewagner/2735af7eea411e18cf20) (Shu-yu Guo)
+1. Do we want to revisit class "mixins" (ie. where the _ClassHeritage_ evaluates to a super class which ` IsConstructor(superclass) === false`)? (Rick Waldron) ([plan](https://gist.github.com/rwaldron/cc607ca68d076c6332f771ac7db17228), [context](https://github.com/rwaldron/tc39-notes/blob/master/es6/2012-07/july-26.md#maxmin-class-semantics), [context](https://github.com/rwaldron/tc39-notes/blob/master/es6/2013-01/jan-31.md#class-extends-throw-on-non-constructor), [relevant spec](https://tc39.github.io/ecma262/#sec-runtime-semantics-classdefinitionevaluation))
+1. Reconsider [arrow generator syntax](https://gist.github.com/threepointone/014954c9270749d0b1d1051c12a705af) per [2013-11-20 meeting notes item 4.10 resolution](https://esdiscuss.org/notes/2013-11-20#4-10-generator-arrow-function-syntax) (Brendan Eich)
+1. [HostObjectDefinePropertyReturnFalse](https://github.com/tc39/ecma262/pull/688) (Domenic Denicola)
+1. [System.global](http://tc39.github.io/proposal-global/) to stage 3 (Jordan Harband)
+1. [Named group captures](https://github.com/littledan/es-regexp-named-groups) (Brian Terlson and Daniel Ehrenberg) for Stage 1 [slides](https://docs.google.com/presentation/d/1b3CigDqepiupv7jQbHyKVkRG72t2qIxeN_DnX75jTY8/edit)
+1. [Intl.Segmenter](https://github.com/littledan/Segmenter) (Daniel Ehrenberg) for Stage 1 [slides](https://docs.google.com/presentation/d/1X2zBU3bZ4ergVMWfubCsdnHFzeaDgqiTRJVgvNGjQBs/edit#slide=id.p)
+1. [Set, Map, WeakSet, and WeakMap: `of` and `from` methods](https://github.com/leobalter/proposal-setmap-offrom) (Leonardo Balter) for Stage 1
+1. Sigil swap: decorators `<->` private fields (Jordan Harband)
+1. [RegExp Unicode Property Escapes](https://github.com/mathiasbynens/es-regexp-unicode-property-escapes) to stage 2 (championed by Brian Terlson and Daniel Ehrenberg, text by Mathias Bynens) [slides](https://docs.google.com/presentation/d/1o31S9RqDdkoWW2zfPMNIZdPDIp25Rr0-XW0gro_cskk/edit)
+1. [OrdinarySetPrototypeOf fix](https://github.com/tc39/ecma262/issues/683) / [Prototype loops and Proxy objects](https://github.com/tc39/ecma262/issues/683)  ([slides](https://docs.google.com/presentation/d/1kHuEtVc-GPp3rbddMVBATKYQ5qLz2o4LnNGUNg8cCz4/edit?usp=sharing)) (Michael Saboff, Saam Barati)
+1. [Cancelable promises](https://github.com/tc39/proposal-cancelable-promises) for stage 2 (Domenic Denicola)
+1. Update on integrating [Observable](https://docs.google.com/presentation/d/18KkpDm0Z-lGnUFxcK_ZJwSKCSalnBqjhGN8W--PyT88/edit?usp=sharing) with Cancel Tokens (Jafar Husain).
+1. [import()](https://github.com/domenic/proposal-import-function) for stage 1 or 2 (Domenic Denicola)
```

Comment:
This PR exists because @jzaefferer asked to include the agenda items in the report, I don't know yet how to do it properly. Maybe I should just import the titles without these further descriptions. It's probably confusing.